pinc/phpstan_exceptions.inc

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Would phpstan let us put this in `SETUP/` to put it with the rest of the dev-centric stuff? Not sure if it'll let us since we're excluding that path.

Either way I'd rename it `phpstan_bootstrap.inc`.

Also please put a newline between the function defs.

Reply:
Good idea. Done.